### PR TITLE
Show missing outline message

### PR DESCRIPTION
**Pull request by Joozty**
@viktor-yakubiv You may use this ID 196558092

### CODE REVIEW

File: reader/components/outline-sidebar/OutlineSidebar.jsx [hunk 1]
```diff
@@ -46,11 +46,20 @@ class OutlineSidebar extends React.PureComponent {
...
-        isExpanded
-        linkService={linkService}
-        outline={finalOutline}
-      />
+      <>
+        {!outline && (
+          <div className="missing-outline">
+            Document outline is not available for this moment. We are working
+            hard to bring this feature to bring this feature to majority of
+            works as soon as possible.
```
Comment (viktor-yakubiv) [inline reader/components/outline-sidebar/OutlineSidebar.jsx]: Let's keep only this:
> Document outline is not available for this moment.

File: reader/components/outline-sidebar/OutlineSidebar.jsx [hunk 2]
```diff
@@ -46,11 +46,20 @@ class OutlineSidebar extends React.PureComponent {
     ]
 
     return (
-      <OutlineGroup
-        isExpanded
-        linkService={linkService}
-        outline={finalOutline}
-      />
+      <>
+        {!outline && (
```
Comment (viktor-yakubiv) [inline reader/components/outline-sidebar/OutlineSidebar.jsx]: Could not it be an empty array? If so, you should also check for a `length`

File: reader/components/outline-sidebar/OutlineSidebar.scss [hunk 3]
```diff
@@ -15,35 +15,48 @@ $expand-icon: url("assets/expand.svg");
...
+      }
+
+      [aria-expanded="false"] > & {
+        transform: rotate(-90deg);
+      }
     }
+  }
+
+  .missing-outline {
+    padding: 0.75rem;
```
Comment (viktor-yakubiv) [inline reader/components/outline-sidebar/OutlineSidebar.scss]: It was 16px, i.e. 1rem everywhere. However, this is also good.

File: reader/components/outline-sidebar/OutlineSidebar.scss [hunk 4]
```diff
@@ -15,35 +15,48 @@ $expand-icon: url("assets/expand.svg");
...
+    border-style: dashed;
+    border-color: #bfbfbf;
```
Comment (viktor-yakubiv) [inline reader/components/outline-sidebar/OutlineSidebar.scss]: Try to use `$gray-*` accent everywhere. This is why we have configuration files.

File: reader/components/outline-sidebar/OutlineSidebar.jsx (hunk 2 above)
Comment (Joozty) [reply reader/components/outline-sidebar/OutlineSidebar.jsx]: This is something I don't know. I will add it there just to be sure.

File: reader/components/outline-sidebar/OutlineSidebar.scss (hunk 4 above)
Comment (Joozty) [reply reader/components/outline-sidebar/OutlineSidebar.scss]: https://github.com/oacore/reader/pull/106/commits/a8d706477d649c2151095655eeaaded63666f0a7

File: reader/components/outline-sidebar/OutlineSidebar.jsx (hunk 2 above)
Comment (Joozty) [reply reader/components/outline-sidebar/OutlineSidebar.jsx]: https://github.com/oacore/reader/pull/106/commits/a8d706477d649c2151095655eeaaded63666f0a7

File: reader/components/outline-sidebar/OutlineSidebar.jsx (hunk 1 above)
Comment (Joozty) [reply reader/components/outline-sidebar/OutlineSidebar.jsx]: https://github.com/oacore/reader/pull/106/commits/a8d706477d649c2151095655eeaaded63666f0a7

**Pull request merged**